### PR TITLE
Assignment 10: complete

### PR DESCRIPTION
**Pull request by enviro3**
-Changed the song position and value to match the song and the song plays
-Volume bar now adjusts 

@droberts84

### CODE REVIEW

File: app/scripts/services/SongPlayer.js [hunk 1]
```diff
@@ -60,6 +60,13 @@
         * @type {Object}
         */ 
         SongPlayer.currentSong = null;
+        
+        /**
+        * @desc volume control
+        * @type {Number}
+        */ 
+        SongPlayer.volume = 80;
```
Comment (drobtravels) [inline app/scripts/services/SongPlayer.js]: Does this ever change?   (Try adding `console.log`s to check if you aren't sure).  Where does it change?

File: dist/scripts/services/SongPlayer.js [hunk 2]
```diff
@@ -35,9 +35,9 @@
             
             currentBuzzObject = new buzz.sound(futureSong.audioUrl, {
                 formats: ['mp3'],
-                preload: true    
+                preload: true
             });
-            
+            currentBuzzObject.setVolume(SongPlayer.volume);
```
Comment (drobtravels) [inline dist/scripts/services/SongPlayer.js:40]: This is a great idea.  What is the purpose of this?

**Review by drobtravels — CHANGES_REQUESTED** [review]

File: app/scripts/services/SongPlayer.js (hunk 1 above)
Comment (enviro3) [reply app/scripts/services/SongPlayer.js]: setVolume does change it later on

File: dist/scripts/services/SongPlayer.js (hunk 2 above)
Comment (enviro3) [reply dist/scripts/services/SongPlayer.js:40]: When a song changes, this piece of code ensures that the new sound file that is played is set to the volume that the last song was playing based on where the volume slider bar is.

File: app/scripts/services/SongPlayer.js (hunk 1 above)
Comment (drobtravels) [reply app/scripts/services/SongPlayer.js]: Does it?  Try `console.log(SongPlayer.volume)` in that funciton

File: dist/scripts/services/SongPlayer.js (hunk 2 above)
Comment (drobtravels) [reply dist/scripts/services/SongPlayer.js:40]: 👍

File: app/scripts/services/SongPlayer.js (hunk 1 above)
Comment (enviro3) [reply app/scripts/services/SongPlayer.js]: Ahh, i see what happened. So the volume was changing but SongPlayer.volume was always 80. I fixed this now

**Pull request merged**